### PR TITLE
release: cut the zone.js-0.11.6 release

### LINKED ISSUE #46230: Missing release of zone.js along with Angular 14.0.0
### Which @angular/* package(s) are the source of the bug

I wouldn’t call it a bug strictly, but those are the related packages:
@angular/core, zone.js

### Is this a regression?

No

### Description

Angular 14.0.0 Changelog [mentions some changes in zone.js](https://github.com/angular/angular/blob/main/CHANGELOG.md#zonejs) (related to this commit https://github.com/angular/angular/commit/f19b36f462803b3b3b9410391c039649541b10bc), however new version of zone.js was not released yet. 

By any chance, do you plan to release new zone.js@0.11.6 (or 0.12.0?) and point it as a peerDependency of angular/core@14.0.1?

If not, I think we should remove the zone.js changes from Angular 14 changelog. 

cc @JiaLiPassion

### Please provide a link to a minimal reproduction of the bug

N/A

### Please provide the exception or error you saw

```true
N/A
```

### Please provide the environment you discovered this bug in (run `ng version`)

```true
Angular 14.0.0
```

### Anything else?

I’m not sure if the issue template for bug is appropriate for this issue, but other templates like “feature request” or “docs” didn’t look more appropriate to me.

### PR DESCRIPTION
**Pull request by JiaLiPassion**
Close #46230

### CODE REVIEW

File: packages/zone.js/CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+The breaking change is scoped only to the plugin
+`zone.js/plugins/task-tracking`. If you used `TaskTrackingZoneSpec` and
+checked the pending macroTasks e.g. using `(this.ngZone as any)._inner
+._parent._properties.TaskTrackingZone.getTasksFor('macroTask')`, then
+its behavior slightly changed for periodic macrotasks. For example,
+previously the `setInterval` macrotask was no longer tracked after its
+callback was executed for the first time. Now it's tracked until
+the task is explicitly cancelled, e.g  with `clearInterval(id)`.
+
+fixes 45350
```
Comment (Platonn) [inline packages/zone.js/CHANGELOG.md:26]:
```suggestion
fixes #45350
```

**Review by alxhub — APPROVED** [review]

File: packages/zone.js/CHANGELOG.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (JiaLiPassion) [reply packages/zone.js/CHANGELOG.md]: @Platonn , got it, I removed the `fixes` part.

**Comment by dylhunn** [comment]
@JiaLiPassion Do you have plans release zone.js soon? (I have reopened #46230 as well.)

**Review by alxhub — APPROVED** [review]

**Comment by jessicajaniuk** [comment]
This PR was merged into the repository by commit 1120d540d863922308c116da0ba33f08ceb87ac1.

**Pull request closed without merging**